utils/canonicalUrls.js

```diff
@@ -71,6 +71,12 @@ export const bestPubUrl = ({ pubData, communityData }, options = {}) => {
 	return pubShortUrl(pubData);
 };
 
+export const pubSettingsUrl = (community, pub, section) => {
```

Comment:
We already have a `getDashUrl` that'll do this for ya. The invocation you want is:

```ts
getDashUrl({ pubSlug: pub.slug, mode: 'settings' })
```